src/Uno.UI.RuntimeTests/Tests/Windows_Data/Pdf/Given_PdfDocument.cs

```diff
@@ -0,0 +1,224 @@
+﻿#nullable enable
+#if !HAS_UNO || XAMARIN_ANDROID
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Private.Infrastructure;
+using Uno.UI.RuntimeTests.Helpers;
+using Windows.Data.Pdf;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_Data.Pdf;
+
+[TestClass]
+[RunsOnUIThread]
+#if HAS_UNO && !XAMARIN_ANDROID
+[Ignore("Not implemented yet.")]
+#endif
+public class Given_PdfDocument
+{
+	private const string PdfDocument_Password_Valid = "uno";
+	private const string PdfDocument_Password_Invalid = "1";
+	private const string PdfDocument_Name = "UnoA4.pdf";
+	private const string PdfDocument_Name_Protected = "UnoA4_Protected.pdf";
+	private const string ReferencePageImageUri = "UnoA4.png";
+	private const string ReferencePageImage_ProtectedUri = "UnoA4_Protected.png";
+
+	private static Stream? GetSreamFromResource(string resourceName)
+	{
+		var assembly = typeof(Given_PdfDocument).Assembly;
+		var resourceFullName = assembly
+			.GetManifestResourceNames()
+			.FirstOrDefault(name => name.EndsWith(resourceName));
+
+		Assert.IsNotNull(resourceFullName, $"Unable to find resource named {resourceName}.");
+
+		return assembly.GetManifestResourceStream(resourceFullName);
+	}
+
+	private static async Task CheckDocumentAsync(PdfDocument? document
+		, string referencePageImage
+		, PdfPageRenderOptions? options = default
+		, bool hasPassword = false)
```

Comment:
I prefer this style because it is clearer when it shows the differences. eg:

```csharp
private static async Task CheckDocumentAsync(PdfDocument? document,
		string referencePageImage,
		PdfPageRenderOptions? options = default,
		bool hasPassword = false)
```
```csharp
private static async Task CheckDocumentAsync(PdfDocument? document,
		string referencePageImage,
		PdfPageRenderOptions? options = default,
		bool hasPassword = false,
                bool extra = true)
```

```diff
private static async Task CheckDocumentAsync(PdfDocument? document,
		string referencePageImage,
		PdfPageRenderOptions? options = default,
-               bool hasPassword = false)
+	        bool hasPassword = false,
+               bool extra = true)
```

```csharp
private static async Task CheckDocumentAsync(PdfDocument? document
		, string referencePageImage
		, PdfPageRenderOptions? options = default
		, bool hasPassword = false
)
```
```csharp
private static async Task CheckDocumentAsync(PdfDocument? document
		, string referencePageImage
		, PdfPageRenderOptions? options = default
		, bool hasPassword = false
                , bool extra = true
)
```

```diff
private static async Task CheckDocumentAsync(PdfDocument? document
		, string referencePageImage
		, PdfPageRenderOptions? options = default
		, bool hasPassword = false
+               , bool extra = true
)
```

in the second case it is immediately identified that only the extra parameter has been added.

I don't know what one's policy is for these situations.

Tell me if I can stay like this or I change it.

Reply:
I can see the benefit of your style. But that's not common and is less consistent with the rest of the codebase.

Reply:
Placing commas is common in other languages, it allows for commenting lines more easily without having to modify the previous one. I don't mind that syntax, using it myself :)